src/domain/search/landingPage/LandingPage.tsx

```diff
@@ -17,9 +17,8 @@ export function LandingPageContentLayout({
 }: LandingPageProps & PageContentLayoutProps) {
   const { title, description, heroLink } = landingPage?.translation || {};
   const heroImage =
-    landingPage?.desktopImage?.edges &&
-    landingPage?.desktopImage?.edges[0]?.node?.mediaItemUrl;
-
+    landingPage?.desktopImage?.edges?.[0]?.node?.mediaItemUrl ?? undefined;
+  console.log('heroImage', heroImage);
```

Comment:
console.log!?

Reply:
We should  have eslint to prevent these. I can add that rule too.